package.json

```diff
@@ -88,6 +88,7 @@
     "WebDriverAgentLib",
     "WebDriverAgentRunner",
     "WebDriverAgentTests",
-    "XCTWebDriverAgentLib"
+    "XCTWebDriverAgentLib",
+    "WebDriverAgentRunner-Runner.app"
```

Comment:
How the package size will be?
I think it's better to make `.app` to `tar` to pack this module when we push this

Reply:
I agree, we don't want hundreds of files being downloaded. Probably need a mechanism to pack/unpack.

Reply:
(zip might make sense since we have code for all that already)

Reply:
Updated. I made it shell out on `zip` from the terminal. It will only work in MacOS but that's fine because we need to be in a MacOS environment to produce the .app